File: C#/Урок 038/Задача 02.cs
class Program
{
    static void Main()
    {
        System.Console.Write("Enter a positive number: ");
        int userNumber = System.Convert.ToInt32(System.Console.ReadLine());

        if (userNumber <= 0)
        {
            System.Console.WriteLine("Your number is not positive");
        }
        else
        {
            int number = 0;
            while (number <= userNumber)
            {
                System.Console.Write(number + " ");
                number++;
            }
        }
    }
}

File: C++/Урок 028 - if/Задача 02.cpp
#include <iostream>

int main()
{
    std::cout << "Enter a number: ";
    int number;
    std::cin >> number;

    if (number > 0)
    {
        std::cout << "Your number is positive\n";
    }
    else
    {
        std::cout << "Your number isn't positive\n";
    }

    return 0;
}
